egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -1011,26 +1011,35 @@ if ! "${skip_eval}"; then
             fi
         fi
 
-        # NOTE(kamo): If feats_type=raw, vocoder_conf is unnecessary
         _scp=wav.scp
         if [[ "${audio_format}" == *ark* ]]; then
             _type=kaldi_ark
         else
             # "sound" supports "wav", "flac", etc.
             _type=sound
         fi
+
+        # NOTE(kamo): If feats_type=raw, vocoder_config is unnecessary
         if [ "${_feats_type}" = fbank ] || [ "${_feats_type}" = stft ]; then
```

Comment:
Since no one use this feat type, I will remove the support of feats_type=fbank/stft in the next PR.